api/adaptive_load/scoring_function.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+
+package nighthawk.adaptive_load;
+
+import "google/protobuf/any.proto";
+
+// Configures a ScoringFunction plugin that measures proximity to a threshold
+// according to an arbitrary formula.
+message ScoringFunctionConfig {
```

Comment:
(optional) Is it worthwhile to create a separate file for each of these SomethingConfig proto messages? Do we expect to be adding more into these files? Or should we just have a config.proto file with all of them?

Reply:
Moved all to config.proto.